mkdocs.yml

```diff
@@ -124,6 +124,7 @@ nav:
       - Overview: panel/webhooks/index.md
       - Examples: panel/webhooks/examples.md
     - Plugins:
+      - 'Custom Links': panel/Plugins/Custom-links.md
```

Comment:
This was missing in the original version of the MR. Without this - the page will not appear in the main nav or the sitemaps.